<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1129</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Grumpy IRC</string>
  </property>
  <property name="windowIcon">
   <iconset resource="embedded.qrc">
    <normaloff>:/icons/img/grumpy_logo.png</normaloff>:/icons/img/grumpy_logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1129</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionOpen_window"/>
    <addaction name="actionPreferences"/>
    <addaction name="actionProxy"/>
    <addaction name="actionEnable_proxy"/>
    <addaction name="separator"/>
    <addaction name="actionConnect"/>
    <addaction name="actionFavorites"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuScrollback">
    <property name="title">
     <string>Scrollbac&amp;k</string>
    </property>
    <addaction name="actionLoad_more_items_from_remote"/>
    <addaction name="actionToggle_secret"/>
    <addaction name="actionExport_to_html"/>
    <addaction name="actionExport_to_plain_text"/>
   </widget>
   <widget class="QMenu" name="menuScripts">
    <property name="title">
     <string>Scripts</string>
    </property>
    <addaction name="actionScript_manager"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuScrollback"/>
   <addaction name="menuScripts"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="statusTip">
    <string/>
   </property>
  </widget>
  <action name="actionConnect">
   <property name="text">
    <string>&amp;Connect</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>-</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionResync">
   <property name="text">
    <string>Resync</string>
   </property>
  </action>
  <action name="actionLoad_more_items_from_remote">
   <property name="text">
    <string>&amp;Load more items from remote</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionOpen_window">
   <property name="text">
    <string>&amp;Open new grumpy window</string>
   </property>
   <property name="shortcut">
    <string>Alt+N</string>
   </property>
  </action>
  <action name="actionFavorites">
   <property name="text">
    <string>&amp;Favorites</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionToggle_secret">
   <property name="text">
    <string>&amp;Secured input</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
  </action>
  <action name="actionProxy">
   <property name="text">
    <string>Proxy</string>
   </property>
  </action>
  <action name="actionEnable_proxy">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable proxy</string>
   </property>
  </action>
  <action name="actionExport_to_html">
   <property name="text">
    <string>Export to html</string>
   </property>
  </action>
  <action name="actionExport_to_plain_text">
   <property name="text">
    <string>Export to plain text</string>
   </property>
  </action>
  <action name="actionScript_manager">
   <property name="text">
    <string>Script manager</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="embedded.qrc"/>
 </resources>
 <connections/>
</ui>
